product_code/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/user/Admin.scala

```diff
@@ -25,16 +27,21 @@ case class Admin(
       phoneNumber: String = this.phoneNumber,
       birthDate: String = this.birthDate
   ): Admin =
-    copy(username, role, address, firstName, lastName, email, phoneNumber, birthDate)
+    copy(username, isActive, role, address, firstName, lastName, email, phoneNumber, birthDate)
 
   override def trim: Admin = super.trim.asInstanceOf[Admin]
 
   override def toPublic: Admin =
-    Admin(this.username, this.role, Address.empty, this.firstName, this.lastName, this.email, this.phoneNumber, "")
+    Admin(this.username, this.isActive, this.role, Address.empty, this.firstName, this.lastName, this.email, this.phoneNumber, "")
 
   //override def toPublic: Admin = super.toPublic.asInstanceOf[Admin]
```

Comment:
I know this is not a mistake on this branch, but this should be removed. We may do that here or after the merge directly on develop